test/passes/sort-blocks.txt

```diff
@@ -0,0 +1,69 @@
+(module
```

Comment:
One advantage of defining a total ordering over instructions instead of hashing would be that these results would look better to a human reader, but of course that's not worth doing extra work for.